specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json

```diff
@@ -0,0 +1,1173 @@
+        "parameters": [
+          {
+            "in": "path",
+            "name": "hub",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$"
+          },
+          {
+            "in": "query",
+            "name": "excluded",
+            "description": "Exclude these connectionIds when closing the connections in the hub.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi"
+          },
+          {
+            "in": "query",
+            "name": "reason",
+            "description": "The reason closing the client connection.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The version of the REST APIs.",
+            "type": "string",
+            "default": "2021-10-01"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Error response",
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/api/hubs/{hub}/:send": {
+      "post": {
+        "tags": [
+          "webpubsub"
+        ],
+        "summary": "Broadcast content inside request body to all the connected client connections.",
+        "operationId": "WebPubSub_SendToAll",
+        "consumes": [
+          "application/octet-stream",
+          "text/plain",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "hub",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$"
+          },
+          {
+            "in": "query",
+            "name": "excluded",
+            "description": "Excluded connection Ids.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The version of the REST APIs.",
+            "type": "string",
+            "default": "2021-10-01"
+          },
+          {
+            "in": "body",
+            "name": "message",
+            "description": "The payload body.",
+            "required": true,
+            "schema": {
+              "format": "binary",
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
It is a fire-and-forget pattern instead of a long-running operation.

Reply:
@vicancy -- If I'm understanding this API correctly, this is the operation that will publish the message to the subscribers. It makes sense where this would be 'fire and forget,' but is there any scenario where the client would need to understand or reason about if the :send completed successfully, is in progress, etc?

Reply:
It is a pub/sub pattern. 202 means the message is successfully published to the channel, but since the service don't know whether the subscribers successfully receive the messages (which requires some ack mechanism from the subscriber which we don't support and don't want to support in this send API) I think 202 makes more sense than 200

Reply:
@vicancy -- In this case, we would still recommend a `202`, but you do not include the `operation-location` header. The `202` is the mechanism the service would use to indicate that it will do its best to deliver the message. That is, it has `Accepted` the message to deliver. A `200` would imply that the message has been delivered.

Reply:
Yes, we will not include `operation-location` header.